bls377/pairing.go

```diff
@@ -49,8 +49,7 @@ func (z *GT) FinalExponentiation(x *GT) *GT {
 	var t [6]GT
 
 	// easy part
-	t[0].FrobeniusCube(&result).
-		FrobeniusCube(&t[0])
+	t[0].InverseUnitary(&result)
```

Comment:
Can you replace `t[0].InverseUnitary(&result)` by `t[0].Conjugate(&result)`? Because at this stage the element is not unitary, we just want to apply the Frob^6, which indeeds acts as a conjugation on Fp12 (I would like to keep `InverseUnitary` for elements which are really unitary, even if at the end it's the same operation). Same for the other curves :)

Reply:
Makes sense :)